.github/CONTRIBUTING.md

```diff
@@ -1,14 +1,11 @@
 ## Guide to Contributing to the Toolbox
 
-### Standards
-- Standard 1
-- Standard 2
-- Standard 3
-
-### Workflows
-- Workflow 1
-- Workflow 2
-- Workflow 3
-
-### Documentation
-- Links to documentation
+### ZenHub
+
+The software lifecycle in Toolbox is tracked using an external tool that sits on top of GitHub called ZenHub. ZenHub can be used from the [webapp](app.zenhub.com) or the [chrome extension](https://chrome.google.com/webstore/detail/zenhub-for-github/ogcgkffhplmphkaahpmffcafajaocjbd?hl=en-US). 
+
+ZenHub adds swim-lane boards which are used to organize issues. GitHub labels help organize issues in the swim-lanes and are required.  
+
+ZenHub also adds Milestones and Epics. Milestones are used for releases or features. Epics track larger development efforts which may span multiple releases.
```

Comment:
Milestones are actually provided by Github in vanilla-Github.  I would still keep how we plan to use them in here though.